astro.config.mjs

```diff
@@ -102,23 +102,6 @@ export default defineConfig({
       ],
     }),
   ],
-  redirects: {
```

Comment:
No longer relevant because we'll have to do the redirects from AWS (most likely)